conf/evolutions/default/111.sql

```diff
@@ -0,0 +1,21 @@
+# --- !Ups
+
+ALTER TABLE patients
+  ADD fuzzy_first_name VARCHAR(255),
```

Comment:
Because there are many different methods of fuzzy matching it may be helpful to label the columns as dm_first_name rather than just fuzzy in case another algorithm is used in the future to generate a different code. The change may not be necessary now though.